README.md

```diff
@@ -29,6 +29,11 @@ Our over-arching goal for TidyTuesday is to provide real-world datasets so that
 - Share your work with the world using the hashtags #TidyTuesday and #PydyTuesday so that Posit has the chance to highlight your work, too!
 - Deploy or share your work however you want! If you'd like a super easy way to publish your work, give [Connect Cloud](https://connect.posit.cloud/) a try.
 
+### TidierTuesday.jl: A *Tidier* library for TidyTuesday
+
+- Exploring the TidyTuesday data in Julia? Tidier has a new package that will help you out! 
```

Comment:
```suggestion
- Exploring the TidyTuesday data in Julia? [Tidier has a package that will help you out](https://github.com/TidierOrg/TidierTuesday.jl)! 
```
I want to keep these instructions evergreen. We can mention that it's new in our manual social media posts and such. I'm going to come back to take a crack at this section overall after the merge, though, because it has "ads" for Posit's Python thing and TidierTuesday.jl now, but the section above those intentionally doesn't mention any specific languages; TidyTuesday has always been aimed at all languages, we just had R instructions because nobody added the other instructions until recently...